instance/github.py

```diff
@@ -86,6 +87,18 @@ def get_settings_from_pr_body(pr_body):
         return ''
 
 
+def get_ephemeral_databases_from_pr_body(pr_body):
+    """
+    Return True if the PR body specified that the sandbox should use ephemeral
+    databases, False if it specifies persistent databases, or None otherwise
+    """
+    if re.search(r'..Sandbox:?...*\(ephemeral databases?\)', pr_body):
```

Comment:
@omarkhan This still doesn't match the specs:

> It should be possible to disable it from a PR description, by adding the string "ephemeral database" on a line also containing the **sandbox domain name** - ie it would for example be enabled by:
> 
> * [LMS](http://sandbox.example.com) (ephemeral database)

https://opencraft.uservoice.com/forums/280754-opencraft/suggestions/9541143-preserve-databases-during-sandbox-reprovisioning

The addition of "persistent databases" is a good idea; on the other side, checking against "sandbox" wouldn't match the line from the example above - plus it adds logic specific to sandboxes again. I know you likely did this because you don't have the instance domain name on the PR object, but this could be passed on as an argument.